ci-operator/templates/openshift/installer/cluster-launch-installer-e2e.yaml

```diff
@@ -818,6 +812,9 @@ objects:
           mkdir -p /tmp/artifacts/must-gather
           queue /tmp/artifacts/must-gather/must-gather.log oc --insecure-skip-tls-verify adm must-gather --dest-dir /tmp/artifacts/must-gather
 
+          echo "Removing ssh-bastion ..."
+          queue /dev/null oc --insecure-skip-tls-verify --request-timeout=5s delete project openshift-ssh-bastion
```

Comment:
do we need to verify that the project is fully removed before we continue?  is it possible that we will leak a loadbalancer otherwise?

Reply:
IMO, `oc delete` waits for the namespace to be fully deleted.